Formula/c/copa.rb

```diff
@@ -1,8 +1,8 @@
 class Copa < Formula
   desc "Tool to directly patch container images given the vulnerability scanning results"
   homepage "https://github.com/project-copacetic/copacetic"
-  url "https://github.com/project-copacetic/copacetic/archive/refs/tags/v0.4.0.tar.gz"
-  sha256 "d12b97f39147c52ae49ef56c439cf2519d11a0de4cbf1af636c156d746149233"
+  url "https://github.com/project-copacetic/copacetic/archive/refs/tags/v0.4.1.tar.gz"
+  sha256 "1aad8715071852f4a3d3950af1ab7db49eef423c5db1f739238415051310ff72"
   license "MIT"
```

Comment:
- https://github.com/project-copacetic/copacetic/pull/304

Reply:
Seems it's not in 0.4.1 yet: https://github.com/project-copacetic/copacetic/blob/v0.4.1/LICENSE

Reply:
make sense.